### ISSUE TITLE
Should verdict assignment depend on response verification?

### ISSUE THREAD

**Issue by jugglinmike** [w3c/aria-at#938]
<details>
  <summary>Context: explanation of the term "verdict"</summary>

Currently, the process of "running a test" involves two steps:

1. collecting AT responses, and
2. interpreting whether the collected AT responses satisfy the given assertions

The result is a series of "verdicts" (we'll be adding a definition along these lines to [the glossary](https://github.com/w3c/aria-at/wiki/Glossary) soon).

Historically, we haven't discussed these as separate operations because we've expected every tester to perform both. It will soon be necessary to consider them as distinct steps because we will support automated AT response collection, but we will *not* support automated verdict assignment.

</details>

Issues gh-936 and gh-937 describe new coordination opportunities which will only be possible once we recognize that "running a test" is a two-step process.

The more we consider "AT response collection" and "verdict assignment" as separate tasks (that is, if we embrace the new possibilities in those issues), the more I wonder whether the two need distinct verification steps, where verdict assignment does not begin until AT responses have been corroborated.

In other words: should we ask Testers to assign verdicts before we're confident in the AT responses which they're interpreting?

**Comment by mcking65** [comment]
> Issues [gh-936](https://github.com/w3c/aria-at/issues/936) and [gh-937](https://github.com/w3c/aria-at/issues/937) describe new coordination opportunities which will only be possible once we recognize that "running a test" is a two-step process.
> 
> The more we consider "AT response collection" and "verdict assignment" as separate tasks (that is, if we embrace the new possibilities in those issues), the more I wonder whether the two need distinct verification steps, where verdict assignment does not begin until AT responses have been corroborated.
> 
> In other words: should we ask Testers to assign verdicts before we're confident in the AT responses which they're interpreting?


The most important word in this question is "depend". Consider what would happen to the human process of reviewing a draft plan if we built in this dependency. The process would be something like:

1. Assign multiple people to response collection
2. Resolve differences in recorded responses.
3. Assign multiple people to response analysis.
4. Resolve differences in verdict assignments.

In practice, even in draft plans, the vast majority of tests do not have conflicts in either responses or verdicts. Thus, creating a dependency could double the overhead of report generation without any clear benefits.

i see huge advantage in reducing the overhead of response collection with automation. One thing we might be able to do is have more people reviewing the test plans because the process would be less time consuming. That would improve the quality control of drafts. I don't see much value in requiring alignment of the recorded responses from all testers before allowing testers to assign verdicts to a response.

**Comment by jugglinmike** [comment]
Thanks @mcking65! I'm going to close this as "not planned" based on your feedback alone, given that we've been encouraging folks via other channels to share their perspective here for over a month.
